server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -230,14 +231,21 @@ pub struct ChangelogRow {
     pub source_site_id: Option<i32>,
 }
 
-#[derive(Default, Clone)]
+#[derive(Default, Clone, Serialize, Deserialize, Debug, TS)]
 pub struct ChangelogFilter {
+    #[ts(optional)]
```

Comment:
I was hoping this would be automatic, maybe it is but i can't find it in ts-rs docs